contracts/curation/Curation.sol

```diff
@@ -463,4 +530,28 @@ contract Curation is CurationV1Storage, GraphUpgradeable, ICuration {
             rewardsManager.onSubgraphSignalUpdate(_subgraphDeploymentID);
         }
     }
+
+    /**
+     * @dev Calculate reserve ratio based on initialization phase
+     * @param _createdAt When NameCurationPool was created
+     * @return Reserve ratio
+     */
+    function _getEffectiveReserveRatio(uint256 _createdAt) private view returns (uint32) {
+        uint32 effectiveReserveRatio = defaultReserveRatio;
+
+        if (block.timestamp <= _createdAt.add(initializationPeriod)) {
```

Comment:
We could store this intermediate calculation in a variable as it is reused in different places `_createdAt.add(initializationPeriod)`

Reply:
When we upgrade the Curation contract with this new implementation, all the existing subgraphs deployments will have createdAt set at 0. Maybe we should add if-clause to return the plain reserve ratio when Subgraph.createdAt = 0